supervision/utils/video.py

```diff
@@ -141,6 +673,7 @@ def _validate_and_setup_video(
     return video, start, end
 
 
+@DeprecationWarning
```

Comment:
let's use `deprecated` from `supervision.utils.internal`. it is currently used in the codebase to mark several deprecations.